task/05-regex-tasks.js

```diff
@@ -116,7 +116,8 @@ function getRegexForSSN() {
  *   'Pa55'.match(validator) => false
  */
 function getPasswordValidator(minLength) {
-   throw new Error('Not implemented');
+   let regEx = '^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[A-Za-z\\d]' + `{${minLength},}$`;
+   return new RegExp(regEx)
 }
```

Comment:
You can write it like return new RegExp('^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[A-Za-z\\d]' + `{${minLength},}$`)